harness/determined/common/util.py

```diff
@@ -58,6 +58,14 @@ def get_container_user_token() -> Optional[str]:
     return os.environ.get("DET_USER_TOKEN")
 
 
+def get_det_username_from_env() -> Optional[str]:
+    return os.environ.get("DET_USERNAME")
```

Comment:
question: why use `DET_USERNAME` here when we already use `DET_USER` in other places?

Reply:
I switched it to ```DET_USER``` and ```DET_PASS```

I made it ```DET_USERNAME``` to avoid conflicting with ```DET_USER``` but it seems that it is much more sensible to just also use ```DET_USER```